src/components/layout/nav/styles/nav-styles.ts

```diff
@@ -0,0 +1,203 @@
+        move ? 'all 0.25s' : has_scrolled ? 'all 0.25s' : 'none'};
+    transform: translateX(
+        ${({ button_ref, hide_signup_login, move, mounted }) => {
+            const ref_base = getBaseRef(button_ref)
+
+            if (hide_signup_login) {
+                return 0
+            } else if (move && !hide_signup_login) {
+                if (ref_base && mounted) {
+                    ref_base.style.opacity = 1
+                }
+                return 0
+            } else {
+                if (ref_base && mounted) {
+                    ref_base.style.opacity = 0
+                    const calculation = ref_base.offsetWidth + 2
+                    return `${calculation}px`
+                }
+                return '300px'
+            }
+        }}
+    );
+
+    > a {
+        pointer-events: ${({ move }) => (move ? 'visible' : 'none')};
+        cursor: ${({ move }) => (move ? 'pointer' : 'default')};
+    }
+
+    @media (max-width: 991px) {
+        display: none;
+    }
+`
+
+export const LogoDescription = styled(Flex)`
+    @media (max-width: 525px) {
+        display: none;
+    }
+`
+
+export const LogoLinkMobile = styled(LocalizedLink)`
+    cursor: pointer;
+    display: none;
+
+    @media ${device.tabletL} {
+        display: block;
+        cursor: pointer;
+        margin-left: 2rem;
+    }
+`
+
+export const LogoLink = styled(LocalizedLink)<LogoLinkProps>`
+    text-decoration: none;
+    max-width: ${({ max_width }) => max_width || '16rem'};
+    width: 100%;
+
+    @media ${device.tabletS} {
+        & svg,
+        .gatsby-image-wrapper {
+            width: 10rem;
+        }
+    }
+    @media ${device.mobileL} {
+        & svg,
+        .gatsby-image-wrapper {
+            width: 12rem;
+        }
+    }
+`
+
+export const NavLink = styled.li<{ margin?: boolean }>`
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin-right: 2.4rem;
+
+    &:last-child {
+        margin-right: 0;
+    }
+
+    @media ${device.laptopM} {
+        margin-right: 1rem;
+    }
+
+    ${({ margin }) => margin && 'margin: 0 4rem;'}
+`
+
+export const Line = styled.div`
+    width: 1px;
+    height: 28px;
+    margin-right: 8px;
+    margin-left: 8px;
+    background-color: var(--color-white);
+`
+
+export const StyledLink = styled(LocalizedLink)`
+    ${SharedLinkStyle}
+`
+
+export const PartnerNavigationBarWrapper = styled.nav`
+    background-color: var(--color-black);
+    height: 7.2rem;
+    width: 100%;
+    position: relative;
+    z-index: 1;
+    @media ${device.tabletL} {
+        height: auto;
+    }
+`
+
+export const PartnerWrapper = styled.div`
+    width: 100%;
+    position: fixed;
+    top: 0;
+    z-index: 100;
+
+    .fresnel-between-start-tabletL {
+        display: none;
+    }
+    @media ${device.tabletL} {
+        .fresnel-between-start-tabletL {
+            display: flex;
+            width: 100%;
+            align-items: center;
+        }
+    }
```

Comment:
Could you help to verify if we still use `Show` components in our Navs? If not, we can remove this.

Reply:
No, we don't use it here anymore